Update login form

ivandurst opened the pull request:
https://trello.com/c/ukvuAosQ/274-create-hpe-internal-login-page
Updates the `Loginform` component in Grommet with modifications in this comp, as per my conversation with @ericsoderberghp and @L0ZZI: https://hp.invisionapp.com/d/main#/console/4594420/100749211/preview

The second part of this PR is at https://github.com/grommet/grommet-hpe/pull/10. This PR does not depend on https://github.com/grommet/grommet-hpe/pull/10.

ericsoderberghp commented on src/scss/grommet-core/_objects.login-form.scss:
```diff
@@ -49,7 +49,9 @@
 
   &__footer {
     margin-top: halve($inuit-base-spacing-unit);
+    display: -webkit-flex;
```
I don't think we need the -webkit- prefix for flexbox stuff anymore.

ericsoderberghp commented:
These changes look essentially good to me. I'd be OK with merging them.

ivandurst commented:
@ericsoderberghp I removed the `-webkit-` prefix from the flex stuff. I also removed all `-webkit-box-shadow`, `-webkit-transition`, and `-webkit-text-decoration` from grommet scss, as it's my understanding that these are no longer necessary (more here: http://shouldiprefix.com/).

I removed "Do not merge" from this PR title, it can be merged whenever ready.
